third_party/nvfuser/csrc/ir_nodes.cpp

```diff
@@ -2915,6 +2962,37 @@ std::string Swizzle2D::toInlineString(int indent_size) const {
 
 NVFUSER_DEFINE_CLONE_AND_CREATE(Swizzle2D)
 
+Resize::Resize(
+    IrBuilderPasskey passkey,
+    IterDomain* out,
+    IterDomain* in,
+    Val* left,
+    Val* right)
+    : Expr(passkey) {
+  addOutput(out);
+  addInput(in);
+  addAttribute(left);
+  addAttribute(right);
+}
+
+std::string Resize::toString(int indent_size) const {
+  std::stringstream ss;
+  ss << "Resize: ";
+  ss << in()->toString();
+  ss << " by " << leftExpand()->toString() << " and "
+     << rightExpand()->toString();
```

Comment:
Should we use `toInlineString` here? For example, if `leftExpand()` is `ceilDiv(5, 3)`,`toString` will just print something like `i123`?